bin/nest-server

```diff
@@ -36,38 +37,39 @@ pid() {
 
 set-gunicorn_opts() {
   # Set opts for gunicorn.
-  GUNICORN_OPTS="--bind ${HOST}:${PORT}"
+  GUNICORN_OPTS="--name nest-server"
```

Comment:
This looks like an unrelated change...  Could you comment why binding is not required anymore for the stdout/status, and where the `--name` is used or what it's used for?

Reply:
Ok, I see your point. I undo the this change.